Automattic/woocommerce-payments
# Sync discount display with server processing

Fixes #8583 

#### Changes proposed in this Pull Request

Sync discount display with how the server actually processes them according to ToS

#### Testing instructions

- Enroll test account in promotion
- Go to `payment->settings` screen
- In payment methods find Card and open tooltip with fees brakedown
- Observe that all but FX fees have the discount applied
- Observe that total is calculated correctly

-------------------


**Post merge**
